imap/cyr_expire.c

```diff
@@ -650,22 +849,20 @@ static int parse_args(int argc, char *argv[], struct arguments *args)
 int main(int argc, char *argv[])
 {
     int r = 0;
-    struct arguments args;
+    struct cyr_expire_ctx ctx;
```

Comment:
i would pre-initialise this like `struct cyr_expire_ctx ctx = {0};` rather than memsetting it later (though it probably makes no difference with optimisations)